blog/_posts/2015-07-21-conferences.md

```diff
@@ -18,77 +18,77 @@ As in the last [two](http://railsgirlssummerofcode.org/blog/conference/) [years]
 <img src="https://pbs.twimg.com/media/BzB2OFAIcAAz8Hw.jpg" alt="CodePadawans">
 <br><font color="grey"><small><i>Last year’s RGSoC team CodePadawans at Arrrrcamp (Image: <a href="https://twitter.com/langziehohr/status/518043950679347201" target="_blank">Anika Lindtner</a>)</i></small></font>
 
-#### Our selected conferences  
-As of now, we have over 30 tickets to give away for 13 conferences located all around the world. We are incredibly grateful to these conferences and their amazing organisers, who put a lot of effort and hard work into making these events happen and into making them accessible to everyone. We’re glad to have them support us in promoting diversity in tech, and they deserve all the love in the world <3 
+#### Our selected conferences
+As of now, we have over 30 tickets to give away for 13 conferences located all around the world. We are incredibly grateful to these conferences and their amazing organisers, who put a lot of effort and hard work into making these events happen and into making them accessible to everyone. We’re glad to have them support us in promoting diversity in tech, and they deserve all the love in the world <3
 
 Thank you to:
 
-[eurucamp](http://2015.eurucamp.org/)  
```

Comment:
and I believe this is the only case where it happens, but I haven't looked through all of the posts/changes.